React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -120,7 +133,12 @@ static void registerPerformanceLoggerHooks(RCTPerformanceLogger *performanceLogg
       case ReactMarker::JS_BUNDLE_STRING_CONVERT_START:
       case ReactMarker::JS_BUNDLE_STRING_CONVERT_STOP:
       case ReactMarker::NATIVE_MODULE_SETUP_START:
```

Comment:
You'll need to move around these cases.

`CREATE_REACT_CONTEXT_STOP`, `JS_BUNDLE_STRING_CONVERT_START` ... are still not used, they should not be executing `[weakPerformanceLogger markStartForTag:RCTPLNativeModuleSetup];`

Reply:
Ahhh yeah, nice catch!

I'll sort it out.